app/assets/javascripts/directives/miq_calendar.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * The 'miq-calendar' anngular calendar directive takes care of:
```

Comment:
Also, if we are creating a new directive, perhaps now is the time to drop the miq prefix from everything (including the attributes)?  I guess my other question is why the attributes are prefixed with miqCal.  Is there some kind of global-ness to angular attributes?